backend/server.js

```diff
@@ -6,8 +6,8 @@ import connectDB from "./config/db.js";
 import { notFound, errorHandler } from "./middleware/error.middleware.js";
 import productRoutes from "./routes/product.route.js";
 import userRoutes from "./routes/user.route.js";
-
-// Load ENV variable to process.env variable
+import  {importData, destroyData} from './seeder.js'
+import cors from 'cors'
```

Comment:
do we really need the `cors` package? If we only need it in the development stage then please move it to `devDependencies` on `package.json`